src/oic/utils/authn/user.py

```diff
@@ -387,6 +388,10 @@ class SymKeyAuthn(UserAuthnMethod):
 
     def __init__(self, srv, ttl, symkey):
         UserAuthnMethod.__init__(self, srv, ttl)
+
+        if symkey is not None and symkey == "":
```

Comment:
> The actual check is even stricter, due to the minimal aes keylength, that is enforced in oic.util.aes.build_cipher. ... Maybe the aes code could provide a method to check

I don't see the connection from the `Provider.symkey` to the `oic.util.aes.build_cipher`? Does one of the provider classes call this function? We put the check here [due to the idea in this comment](https://github.com/OpenIDC/pyoidc/pull/324#pullrequestreview-31133941).